src/components/Book.jsx

```diff
@@ -19,19 +23,25 @@ function Book(props) {
   };
 
   return (
-    <div>
+    <div className="book-container">
       <div className="book">
         <div className="book-info">
+          <h3>{category}</h3>
           <h2>{title}</h2>
           <h3>{author}</h3>
-          <h3>{category}</h3>
         </div>
         <div className="btn-control">
-          <button type="button" onClick={bookRemove}>
+          <button className="btn-cl" type="button">Comments</button>
+          <button className="btn-cl" type="button" onClick={bookRemove}>
```

Comment:
- [ ] You almost finish the style. Kindly remove the background color and make the page fullscreen 
![image](https://user-images.githubusercontent.com/38635158/185438885-ad4d6d60-e971-4d69-80eb-040fc26742b1.png)
___
ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/185439261-0c99f949-bbac-4ddf-b4f7-70955adebdaa.png)

Reply:
thanks @ahmetbozaci  my bad i fix it